src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -22,7 +22,7 @@
     get_type_hints,
 )
 
-import cloudpickle
+import cloudpickle  # type: ignore
```

Comment:
Cloudpickle lacked type information so ignoring it so mypy is queit.

Reply:
We could use something like:
```
[mypy-cloudpickle.*]
ignore_missing_imports = True
```
in `mypy.ini`